### PR TITLE
Adjust info screen to announce new Release-Features for 3.1

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Adjust Info Screen to announce new Release-Features for 3.1

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14606

### CODE REVIEW

**Review by KathrinLuecke — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -1839,25 +1839,13 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
...
-"NewVersionFeature_30_selftest_warn_others_description" = "Sie können jetzt mit positiven Ergebnissen von alleine durchgeführten Selbsttests eine Warnung an andere Nutzer der Corona-Warn-App schicken.";
-
-"NewVersionFeature_30_warning_unregistered_tests_title" = "Warnung mit unregistrierten Tests";
-
-"NewVersionFeature_30_warning_unregistered_tests_description" = "Sollten Sie bei einer Teststelle mit einem nicht in der Corona-Warn-App registrierten Schnell- oder PCR-Test positiv getestet worden sein, können Sie jetzt auch mit diesen Testergebnissen andere Nutzer der App warnen.";
-
-"NewVersionFeature_30_tan_hotline_removal_title" = "TAN Hotline entfällt";
-
-"NewVersionFeature_30_tan_hotline_removal_description" = "Die Möglichkeit, eine TAN über die TAN-Hotline zu erhalten, entfällt in einigen Wochen. Sie können nun ohne TAN andere Nutzer der Corona-Warn-App warnen.";
+/* Version 3.1 */
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1848]: This line can be removed.

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift [hunk 2]
```diff
@@ -1282,13 +1282,7 @@ enum AppStrings {
...
 		static let aboutAppInformation = NSLocalizedString("NewVersionFeatures_Info_about_abb_information", comment: "")
 		
-		/* Version 3.0 */
-		static let feature30SelfTestWarnOthersTitle = NSLocalizedString("NewVersionFeature_30_selftest_warn_others_title", comment: "")
-		static let feature30SelfTestWarnOthersDescription = NSLocalizedString("NewVersionFeature_30_selftest_warn_others_description", comment: "")
-		static let feature30WarningUnregisteredTestsTitle = NSLocalizedString("NewVersionFeature_30_warning_unregistered_tests_title", comment: "")
-		static let feature30WarningUnregisteredTestsDescription = NSLocalizedString("NewVersionFeature_30_warning_unregistered_tests_description", comment: "")
-		static let feature30TANHotlineRemovalTitle = NSLocalizedString("NewVersionFeature_30_tan_hotline_removal_title", comment: "")
-		static let feature30TANHotlineRemovalDescription = NSLocalizedString("NewVersionFeature_30_tan_hotline_removal_description", comment: "")
+		/* Version 3.1 */
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:1285]:
```suggestion
```

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1848]: This line can be removed.

```suggestion
```

**Review by naveeddotio — CHANGES_REQUESTED** [review]
Please remove extra lines

**Review by puneetmahali1 — APPROVED** [review]
LGTM! Some minor comments as Naveed mentioned.

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1848]: this is just to keep the convention, this way for 3.2 we know where to add the lines, what do you think?

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**